Common/EnvironmentVariables.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Extensions.Configuration;
+
+namespace Common
+{
+    public class EnvironmentVariables
+    {
+        public static string GetVariable(string variablename)
+        {
+            try
+            {
+                var directory = Directory.GetCurrentDirectory();
+                var secretConfig = new ConfigurationBuilder()
+                    .AddJsonFile($"{directory}/../Shell/variables.json", false, true)
+                    .Build();
+                return secretConfig[variablename];
+            }
+            catch (System.Exception)
+            {
+                throw new NotImplementedException($"{variablename} was not found in variables.json");
```

Comment:
That's not the right exception....create a custom exception for variable not found